src/index.html

```diff
@@ -12,11 +12,50 @@
     />
     <title>Moyo header</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./style.css"
+    />
+    <link
+      rel="stylesheet"
+      href="style.css"
```

Comment:
Keep your attributes correctly formatted. The `href` attribute should start on a new line with 2-space indentation related to the tag.